packages/services/src/models/notification-scanners/NotificationScannerBaseModel.js

```diff
@@ -16,19 +16,29 @@ export default class NotificationScannerBaseModel {
   get _DAYS() { return DAYS }
   get _CLIENT_URL() { return CLIENT_URL }
 
-  // mandatory methods and functions for extended models
-  async scan() { 
-    return [
-      { 
-        address: null,
-        details: {
-          emailTemplateModel: {}
-        }
-      }
-    ]
-  }
-  get emailTemplateAlias() { return null }
-  get scanPeriod() { return 0 * this._DAYS }
+  /**
+   * Each scan must return an object with address and details with optional emailTemplateModel:
+   * return [
+   *   { 
+   *     address: null,
+   *     details: {
+   *       emailTemplateModel: {}
+   *     }
+   *   }
+   * ]
+   */
+  async scan() { throw 'subclass responsibility' }
```

Comment:
This works, but it's generally better to wrap it in a `new Error()` so the stack trace and etc. get included.

Reply:
Yup, let's always use `throw Error` which I think it's the shortcut for `throw new Error()`

Reply:
Yup, let's always use `throw Error` which I think it's the shortcut for `throw new Error()`

Reply:
Added!